node/_crypto/randomFill.ts

```diff
@@ -0,0 +1,60 @@
+import randomBytes from "./randomBytes.ts";
+import { Buffer } from "../buffer.ts";
+
+export default function randomFill(
+  buf: Buffer,
+  cb: (err: Error | null, buf?: Buffer) => void,
+): void;
+export default function randomFill(
+  buf: Buffer,
+  offset: number,
+  cb: (err: Error | null, buf?: Buffer) => void,
+): void;
+export default function randomFill(
+  buf: Buffer,
+  offset: number,
+  size: number,
+  cb: (err: Error | null, buf?: Buffer) => void,
+): void;
+
+export default function randomFill(
+  buf: Buffer,
+  // deno-lint-ignore no-explicit-any
+  offset?: any,
+  // deno-lint-ignore no-explicit-any
+  size?: any,
```

Comment:
I believe these could be strongly typed without using `any`.

Would `number | (err: Error | null, buf?: Buffer) => void` work?

Reply:
Now is better?